test/controllers/graphql_controller_v2_test.rb

```diff
@@ -153,7 +153,7 @@ def setup
                                 }' }
       assert_equal(200, response.status)
       json = JSON.parse(response.body)
-      assert json['data']['search']['records'].first['contributors'].any? { |c| c.has_value? 'Moon, Intae' }
+      assert json['data']['search']['records'].first['contributors'].any? { |c| c.value? 'Moon, Intae' }
```

Comment:
I'm assuming that changes like this are about rubocop flagging `.has_value` as not preferred, rather than some aspect of the index value? I don't object to the change, just wanting to make sure I'm not missing some ramfication to the ticketed change.

Reply:
Yeah, this was just rubocop fixes. I ran it on any file I touched and it fixed a few things out of scope of my changes but it seemed fine to include them. I should have made a note about that though in side effects to clarify the things that were not really part of this work that slid in.